android-lib/src/androidTest/java/com/kinvey/androidTest/store/data/cache/CacheAggregationTest.java

```diff
@@ -56,7 +56,7 @@ public void testCount() throws InterruptedException, IOException {
 //        testManager.login(USERNAME, PASSWORD, client);
         User user = new User();
         user.setId("testId");
-        Client.sharedInstance().setActiveUser(user);
+        Client.sharedInstance.setActiveUser(user);
```

Comment:
wondering why this changed to be a property instead of a method

Reply:
rolled back to use method for `Client` shared instence: `Client.sharedInstance()`